digitaltwin/Samples/device/EnvironmentalSensorSample/ModelDefinitionInterface.cs

```diff
@@ -38,11 +49,11 @@ protected override async Task<DigitalTwinCommandResponse> OnCommandRequest(Digit
                 string commandPayload = JsonConvert.DeserializeObject<string>(commandRequest.Payload);
                 if (commandPayload.Equals(EnvironmentalSensorInterface.EnvironmentalSensorInterfaceId))
                 {
-                    return new DigitalTwinCommandResponse(StatusCodeCompleted, this.environmentalSensorModelDefinition);
+                    return Task<DigitalTwinCommandResponse>.Factory.StartNew(() => new DigitalTwinCommandResponse(StatusCodeCompleted, this.environmentalSensorModelDefinition));
```

Comment:
This should just be:

return Task.FromResult(new DigitalTwinCommandResponse(StatusCodeCompleted, this.environmentalSensorModelDefinition));

Reply:
Sure thing